Session.js

```diff
@@ -113,6 +113,33 @@ module.exports = class Session {
     return allCandidates[0];
   }
 
+  /**
+   * conditionalDecide
+   * Decision option to determine decision to take.
+   *
+   * @param  {string} name - name of event
+   * @param  {array} contexts - contexts to choose from
+   * @param  {Object} options (optional) - timeout
+   * @param  {Function} cb - error and decision
+   */
+  conditionalDecide(name, contexts = [], options = {}, cb) {
+    options.timeout = options.timeout || this.timeout;
+    options.url = this.amp.domain + this.amp.apiPath + this.amp.key + "/conditionalDecide";
+
+    if (utils.isFunction(arguments[arguments.length - 1])) cb = arguments[arguments.length - 1];
+
+    this.request({
+      event: name,
```

Comment:
fill in key, sessionId, userId, decision just like the regular decide request

Reply:
So the request that Yuval sent isn't complete?

Reply:
What @yuval-si sent embeds the EventRequest - so it has everything that the regular decide has
```type ConditionalEventRequest struct {
    Req      *EventRequest            `json:"request"`
    Event    string                   `json:"event"`
    Contexts []map[string]interface{} `json:"contexts"`
}```